apis/actions.md

```diff
@@ -0,0 +1,125 @@
+---
+description: Creating tool items, GLib.Actions, and keyboard shortcuts
+---
+
+# Actions
+
+GTK and GLib have a powerful API called [`GLib.Action`](https://valadoc.org/gio-2.0/GLib.Action.html) which can be used to define the primary actions of your app, assign them keyboard shortcuts, and tie them to [`Actionable`](https://valadoc.org/gtk+-3.0/Gtk.Actionable.html) widgets in your app like Buttons and Menu Items. In this section, we're going to create a Quit action for your app with an assigned keyboard shortcut and a Button that shows that shortcut in a tooltip. Begin by creating a `Gtk.Application` with a `Gtk.ApplicationWindow` as you've done in previous examples.
+
+## Gtk.HeaderBar
+
+Let's begin by creating a new [`Gtk.HeaderBar`](https://valadoc.org/gtk+-3.0/Gtk.HeaderBar.html). Typically your app will have a HeaderBar at the top of the window that will contain tool items, which users will interact with to trigger your app's actions.
```

Comment:
(1) This is the second "begin" in a row - we have already "begun" by creating the Gtk.Application.  Maybe "Now let's add a headerbar to the application ...."

(2) "at the top of the window, which will contain ...."  (it is the HeadBar that contains, not the window so need comma).

(3) "tool items with which users will interact ..." (in order to avoid two ", which .." in a row.  (Or split into two sentences - this kind of sentence might not be clear to non-English speakers)